### ISSUE TITLE
Command line: export a project with a default audio settings

### ISSUE THREAD

**Issue by anytizer** [LMMS/lmms#6735]
Command line export of a project file into different audio settings, based on file extension.

```
lmms export project.mmpz project.ogg
lmms export project.mmpz project.mid
```

Will this be useful?

I believe at least 2 benefits of it, if implemented. First, project conversion is made easy without loading loading the GUI. It might be used to automate the conversion/rendering process. Second, it opens opportunities of live-editor for the web to render a .mmpz file.

For example, there are several [projects](https://lmms.io/lsp/?action=browse&category=Projects&sort=date) that could be converted into automatic audio "preview".

**Comment by zonkmachine** [comment]
I'm confused now. We already have command line export of projects. I can already select an output file with an extension, wav/flac/ogg/mp3.

Example:
`~/builds/lmms/build$ ./lmms render ../data/projects/demos/Skiessi/Skiessi-Onion.mmpz -o ~/Desktop/onion.ogg`

**Comment by anytizer** [comment]
Thanks to point out the render mode. Where can we look for the details? I was actually looking for exactly-similar/this thing.

**Comment by zonkmachine** [comment]
I don't think you can render a midi project from the command line but I'm not sure.

Closing. The documentation needs to be improved. I don't know if some of it got lost when the docs were moved to a new platform.

**Comment by zonkmachine** [comment]
I'd tag this `documentation` if there was such a tag.
